lib/screens/explore_screens/unplant_seeds/interactor/viewmodels/unplant_seeds_bloc.dart

```diff
@@ -12,7 +13,8 @@ import 'package:seeds/screens/explore_screens/unplant_seeds/interactor/viewmodel
 import 'package:seeds/screens/explore_screens/unplant_seeds/interactor/viewmodels/unplant_seeds_state.dart';
 
 class UnplantSeedsBloc extends Bloc<UnplantSeedsEvent, UnplantSeedsState> {
-  UnplantSeedsBloc(RatesState rates) : super(UnplantSeedsState.initial(rates));
+  UnplantSeedsBloc(RatesState rates)
+      : super(UnplantSeedsState.initial(rates, remoteConfigurations.featureFlagDelegateEnabled));
```

Comment:
featureFlagClaimUnplantedSeedsEnabled or featureFlagDelegateEnabled

Just checking.,.

Reply:
Nice catch Thanks, is featureFlagClaimUnplantedSeedsEnabled. Fixed